packages/wallets/src/hooks/useIsRtl.ts

```diff
@@ -0,0 +1,8 @@
+import { getInitialLanguage } from '@deriv-com/translations';
+
+const useIsRtl = () => {
+    const i18nLanguage = getInitialLanguage();
```

Comment:
yep this looks cleaner 👍 updated!

Reply:
After rechecking the implementation I'm reverting this back to use i18nLanguage instead of currentLang as currentLang does not always get immediately updated on language switch (example as can seen in the arrow directions below):
## CurrentLang:

https://github.com/user-attachments/assets/ac2e2a85-66f5-4bb5-8208-23b29fcb15a4

## i18nLanguage:

https://github.com/user-attachments/assets/1d9ff2ab-f3dd-469b-b304-6732a9df93f0

cc: @wojciech-deriv